docs/reference/requirements.md

```diff
@@ -0,0 +1,7 @@
+# Requirements 
+
+- OS Ubuntu 22.04.
+- 1GB of HDD.
+- 2 CPU threads per host.
+- Access to the internet for downloading the [Canonical Spark Image](https://hub.docker.com/r/dataplatformoci/spark/tags).
```

Comment:
It would be good to add a proxy configuration guide in the future

Reply:
and a firewall guide for that matter

Reply:
yes, indeed. I suppose this will become relevant as we introduce security